releasenotes/notes/feature-support-calibrations-roundtrip-47f09bd9ff803479.yaml

```diff
@@ -0,0 +1,32 @@
+---
+features:
+  - |
+    JSON data format is supoorted for saving :class:`.Calibrations` instance.
+    This leverages a custom JSON encoder and decoder to serialize 
+    the entire calibration data including user provided schedule templates.
```

Comment:
Do we need to worry about the case where we initialize from a library or is this already covered here?

Reply:
No. We don't serialize library itself. It directly serializes library pulses, i.e. library doesn't provide version information and we cannot guarantee data compatibility across different environment.